osi_environment.proto

```diff
@@ -129,7 +129,7 @@ message EnvironmentalConditions
     // contrast of a distant object against its background.
     //
     // \par References:
-    // - [2] SHEPARD, Frank D. Reduced visibility due to fog on the highway.
+    // - [2] SHEPARD, Frank D. <em>Reduced visibility due to fog on the highway.</em>
```

Comment:
Should the bullet point here be removed too like in the example in comment.rst?

Reply:
I am not done with the  .proto files, so yes, I will remove the bullet points.